app/konnect/getting-started/configure-mesh.md

```diff
@@ -79,6 +66,28 @@ You can see the services the Kubernetes demo app added by navigating to **Mesh M
 
 For more information about the Kubernetes demo app, see [Explore {{site.mesh_product_name}} with the Kubernetes demo app](/mesh/latest/quickstart/kubernetes/).
 
+## Configure `kumactl` to connect to your global control plane
+
+`kumactl` is a CLI tool that you can use to access {{site.mesh_product_name}}. It can create, read, update, and delete resources in {{site.mesh_product_name}} in Universal/{{site.konnect_short_name}} mode.
+<!--* (not sure if I removed the right line, so keeping in for easy swap 
+Perform read-only operations on {{site.mesh_product_name}} resources on Kubernetes. -->
+
+You connect `kumactl` to the global control plane in {{site.konnect_short_name}} so that you can run commands against the control plane.
+
+1. From the left navigation menu in {{site.konnect_short_name}}, open {% konnect_icon mesh-manager %} [**Mesh Manager**](https://cloud.konghq.com/mesh-manager) and select the `example-cp` control plane.
+1. Select **Configure kumactl** from the **Control Plane Actions** dropdown menu and follow the steps in the wizard to connect `kumactl` to the control plane.
+1. Verify that the services you added from the previous section with the Kubernetes demo app are running correctly:
+```bash
+kumactl get dataplanes
+```
+If your dataplanes were configured correctly with the demo app, it should return something like:
+```bash
+MESH   NAME   TAGS   ADDRESS   AGE
+?
```

Comment:
The output for this command is actually not great today, I have an open issue to track. For now maybe let's just delete the `something like:` and following bash section and just say something like "you should see all 4 dataplanes show in the listed tabular output" or similiar?